File: despatch.Domain/Models/Default/Response/UserAddressResponse.cs
namespace despatch.Domain.Models.Default.Response
{
    public class UserAddressResponse : BaseEntityResponse
    {
        public string UserId { get; set; }
        public string AddressId { get; set; }

        public virtual UserResponse User { get; set; }
        public virtual AddressResponse Address { get; set; }
    }
}

File: despatch.WEB/app/src/swagger/models/user-address-response.ts
/* tslint:disable */
import { BaseEntityResponse } from './base-entity-response';
import { AddressResponse } from './address-response';
import { UserResponse } from './user-response';
export interface UserAddressResponse extends BaseEntityResponse{
  address?: AddressResponse;
  addressId?: string;
  user?: UserResponse;
  userId?: string;
}
